components/measurement/nettests/WebConnectivity.js

```diff
@@ -575,13 +561,7 @@ const WebConnectivityDetails = ({
                       <Flex key={index}>
                         <Box>
                           <Text>
-                            <FormattedMessage
-                              id='Measurement.Details.Websites.TCP.ConnectionTo'
-                              values={{
-                                destination: <strong> {connection.destination} </strong>,
-                                connectionStatus: intl.formatMessage(messages[`connection.${connection.status}`])
-                              }}
-                            />
+                            <strong>{connection.destination}</strong>{connection.failure && `: ${connection.failure}`}
```

Comment:
I suggest we rephrase this as:
```
{destination}: {connectionStatus} (`{failureString}`)
```
Where `connectionStatus` is `failed` or `succeeded` and failure string parens only appears when it's failed and contains the value of `failure`